on_node_start.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+
+# https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425#file-bash_strict_mode-md
+set -euo pipefail
+
+. /etc/parallelcluster/cfnconfig
+
+echo "Node type: ${cfn_node_type}"
+
+function configure_yum() {
+    cat >>/etc/yum.conf <<EOF
+assumeyes=1
+clean_requirements_on_remove=1
+EOF
+}
+
+function install_fuse_overlayfs() {
+    yum -q install buildah
+
+    if [[ ! -e /dev/fuse ]]; then
+        mknod /dev/fuse -m 0666 c 10 229
+    fi
+
+    pushd "$(mktemp -d)"
+    git clone --depth 1 https://github.com/containers/fuse-overlayfs.git ./overlay
+    pushd ./overlay
+    buildah bud -v "$PWD:/build/fuse-overlayfs" -t fuse-overlayfs -f ./Containerfile.static.ubuntu .
+    cp fuse-overlayfs /usr/bin/
+
+}
+
+function yum_cleanup() {
+    yum -q clean all
+    rm -rf /var/cache/yum
+}
+function install_podman() {
+    yum -q install fuse-overlayfs slirp4netns podman
+}
+
+function install_head_node_dependencies() {
+    # MariaDB repository setup
+    curl -sS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | bash
+
+    yum_cleanup
```

Comment:
*query:* I thought you were moving to an Ubuntu-based AMI? Or is that in a later PR?

Reply:
Yeah, next PR, I moved over to ubuntu on Friday I think, so few days after this PR